templates/snmptrapd.sysconfig-Debian.erb

```diff
@@ -0,0 +1,11 @@
+###
```

Comment:
Accordingly with our review guide line, epp is preferred.
https://voxpupuli.org/docs/#reviewing-a-module-pr

Are you able to change the format ?

Reply:
Especially given how all the other templates are .erb, I don’t think this should be a blocker.

Reply:
I wrote it in a erb because the rest of templates are erb. Moreover it is used as:
```puppet
file { 'snmptrapd.sysconfig':
  ensure  => $file_ensure,
  mode    => '0644',
  owner   => 'root',
  group   => 'root',
  path    => $snmp::params::trap_sysconfig,
  content => template($template_snmptrapd_sysconfig),
  require => Package['snmpd'],
  notify  => Service['snmptrapd'],
}                                      
```
where `$template_snmptrapd_sysconfig` is initialized with the corresponding path.

To change it for an epp, all templates should be changed.

Reply:
I was following the guideline, as this template file was not already existing in the module before this PR. So the guideline needs probably to be clarified.